#+TITLE: Research
#+AUTHOR: Patrick O'Neill
#+EMAIL: pon2@umbc.edu
#+SETUPFILE: templates/level0.org
#+INCLUDE: templates/level0.org

* Interests
  
  Transcriptional regulation, biophysics

* Research Summary

  Cells are chemical computers that accept environmental signals as
  inputs and express genes as outputs.  An even shorter way of putting
  this is just to say that /cells compute their metabolism/.  I'm
  interested in helping to understand this process through
  applications of information theory, machine learning and statistical
  physics to the study of transcriptional regulation in bacteria.

  I'm particularly interested in the relationships between the
  parameters that characterize transcription factors and their cognate
  binding sites.  Each regulatory system consists of a transcription
  factor, present at a given mean copy number, and a set of binding
  sites which must be bound with given target probabilities.  For a
  given functional specification there are often many "programs" which
  can encode roughly equivalent behaviors-- how does nature decide?
  From a collection of binding sites known to be co-regulated by a
  transcription factor, what can be inferred about the system
  requirements of the network it participates in and the evolutionary
  dynamics which produced it?  These questions lead us to develop
  mathematical and computational models of protein-DNA co-evolution in
  order to understand the design constraints that gave transcriptional
  regulatory networks their present wiring.
* Publications
  - O'Neill, P.K., Forder, R.K. and Erill, I. "Informational Requirements for Transcriptional Regulation".  To appear in /Journal of Computational Biology/.
  - Joseph P. Cornish, Neus Sanchez-Alberola, Patrick K. Oâ€™Neill, Ronald O'Keefe, Jameel Gheba, and Ivan Erill. "Characterization of the SOS meta-regulon in the human gut microbiome." /Bioinformatics/ first published online January 8, 2014. [[http://bioinformatics.oxfordjournals.org/citmgr%3Fgca%3Dbioinfo%3Bbtt753v2][doi:10.1093/bioinformatics/btt753 ]]
  - O'Neill, P.K., Or, M. and Erill, I. (2013). "Genome-wide Conservation of Codon Usage Bias in fast-growing Bacteria".  /PLoS ONE/. 2013.  [[http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0076177][DOI: 10.1371/journal.pone.0076177]].

  - Kolesnikov, A., Kumchev, A., Howe, D., O'Neill, P. and Tiger, M. (2011). "Estimation of the Commodity Flow of Chlorine from Storage Data".  /J. Transp. Sec./  DOI: 10.1007/s12198-011-0081-4.  
* Talks and Posters
  - "Exact Fermi-Dirac Statistics for Bacterial Transcription Factors". Poster presented to GLBIO 2013.
  - "Exact solutions for transcription factor binding distributions".  Contributed talk for Frontiers in Systems and Synthetic Biology '13.  2013
  - "Genome-wide Conservation of Codon Usage Bias in fast-growing Bacteria".  Poster presented to the Rocky Mountain Bioinformatics Conference.  2012.
  - "Analysis of pair-wise correlations in prokaryotic transcription factor binding motifs".  Poster presented to the 8th International Conference on Bioinformatics.  2011.
  - "Circular reasoning: Stochastic optimization in the disk-covering problem" with Dennis Howell and Matthew Tiger.  Presented to the Towson University Mathematics Colloquium.  2011.
#  - "Finite Splicing Systems".  Presented to the UMBC-UBM Seminar.  2011.
  - "Finite Splicing Systems, Logically and Algebraically".  Presented to the George Washington University Logic Seminar.  2011.
  - "DIY-GEM: tools for citizen scientists", describing joint work with the Baltimore-US iGEM team. Presented to the MIT iGEM Jamboree.  2010.
  - "Silence of the \lambda's: A Geographical Model of Criminal Behavior", with Sara Miller and Dennis Howell. Invited talk for the MD-DC-VA Section MAA Meeting at Virginia State University.  2010.
  - "Risk Estimates for Large-Scale Chlorine Road Transport Networks." Contributed talk for the George Mason University Conference on Atmospheric Transport and Dispersion Modeling. 2010.

